hatch/src/cli/commands/build.rs

```diff
@@ -0,0 +1,64 @@
+use HatchResult;
+use clap::{ App, SubCommand, Arg, ArgMatches };
+use cli::commands::Command;
+use cli::commands::ops::ProjectOps;
+use yaml;
+use project::Project;
+use hatch_error::HatchError;
+
+struct AmbiguousBuilder;
+struct ExplicitBuilder;
+
+impl ProjectOps for AmbiguousBuilder {
```

Comment:
What do you mean by Ambiguous here?  What is the main behavior differences between the AmbiguousBuilder and ExplicitBuilder?

Reply:
After our discussion we are going to use the word Inferred instead of Ambiguous.